packages/lesswrong/components/posts/TableOfContents/ToCColumn.tsx

```diff
@@ -131,25 +159,44 @@ export const ToCColumn = ({tableOfContents, header, welcomeBox, children, classe
         [classes.sideCommentsActive]: sideCommentsActive,
       }
     )}>
+      {hideable &&
+        <div
+          onClick={() => setHidden(!hidden)}
+          className={classNames(classes.hideTocButton, {
+            [classes.hideTocButtonHidden]: !isScrolled,
```

Comment:
This just exists for the case where you scroll up to the top of the page really fast such that the header moves a couple of inches down the page before springing back up. This just stops the button from being visible above the header when that happens. It could be that that just doesn't matter though and this is overkill.